### DISCUSSION TITLE
500 internal server error

### DISCUSSION THREAD

**Discussion by AScience** [jilleb/MQB-FPA#102] [Q&A]
Hi all, not sure if its me doing something wrong but ive checked my file and it seems to be the right format and such, but when i try and upload it goes to a blank page with the following

"Internal Server Error
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application."

I tried the original unmodifed file (converted to txt for this post) which is K19_Param.... which of course doesnt work as its too big of a file with all the extra bits.
So i modified a file called FPA_DATASET and when this is uploaded it causes the error above. 

Any suggestions?

[K19_ParametrizeData_3Q0907530G_5304_TMBKD7NP0J7516248.txt](https://github.com/jilleb/MQB-FPA/files/14470600/K19_ParametrizeData_3Q0907530G_5304_TMBKD7NP0J7516248.txt)

[FPA_DATASET_ODIS_MJ00_240302191328.txt](https://github.com/jilleb/MQB-FPA/files/14470601/FPA_DATASET_ODIS_MJ00_240302191328.txt)

**Comment by jilleb** [comment]
This error occurs when something isnt right in the file format. I see its MJ00, you can pick this one from the dropdown list.

I'll have a look at the files you supplied to see what's wrong.

**Reply by AScience** [reply]
Sorry the reason it’s MJ00 is because I downloaded a random datsset from the editor to inject mine into.

I think it may be me doing something wrong as the start address for mine is not 0B80 like it should be and I’m just putting mine in the 0B80 address as the amount of bytes matches correctly but the start address doesn’t.

(probably me using the system wrong rather than anything else) 

help is very appreciated though :)
